src/api/hooks.ts

```diff
@@ -25,3 +26,30 @@ export function useGetBuildingData(bbl: string): BuildingInfoSWRResponse {
     error: error,
   };
 }
+
+type Tenants2SWRResponse = {
+  data: GCEUser | undefined;
+  error: Error | undefined;
+  isMutating: boolean;
+  reset: () => void;
+  trigger: TriggerWithArgs<GCEUser, unknown, "/gce/upload", GCEPostData>;
+};
+
+export function useSendGceData(): Tenants2SWRResponse {
+  const { data, error, trigger, reset, isMutating } = useSWRMutation(
+    "/gce/upload",
+    Tenants2ApiFetcher,
+    {
+      populateCache: (result: GCEUser) => result,
+      revalidate: false,
+    }
+  );
+
+  return {
```

Comment:
interesting, this is a new pattern to me because I haven't used SWR until GCE. do you ever use `reset` or `isMutating`? just curious

Reply:
yeah new to me as well. And, no, I don't think we'll need those for now. I just saw that SWR had an option for post requests and figured I would stick with it since we were already using it, and this was mostly lifted from the docs.